### PR TITLE
CHE-2015 Rename configuration properties

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?

Rename configuration properties to user-friendly ones.
Add aliases file to support old user configuration
### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/2015

### CODE REVIEW

**Review by skabashnyuk — APPROVED** [review]

**Review by vzhukovs — CHANGES_REQUESTED** [review]
OpenShift plugin need also to be updated with this changes

**Comment by mkuznyetsov** [comment]
OpenShift plugin has its own PR https://github.com/codenvy/plugin-openshift/pull/39

**Review by vzhukovs — APPROVED** [review]

**Comment by TylerJewell** [comment]
@mkuznyetsov @skabashnyuk - I have a couple requests before we merge this PR.  First, it's good work.  I know that there are a lot of details to work through and it was not simple.  I would like this PR to be merged at the same time that we merge:

https://github.com/eclipse/che/pull/2786, which will need to merge and push to Eclipse:

https://github.com/eclipse/che-dockerfiles/pull/29, which will then also require testing of the Che CLI and verification that the docs are current.

If we are going to make a bunch of properties changes like this, we need to end to end test che.properites with changes to che-server, che-launcher and CLI, so that if we break something, which is likely, that we can quickly fix it.  These changes span multiple repositories so while I have tried to test everything locally, it is best for us to merge it all at once, and then to have a very aggressive testing cycle around it.

**Comment by mkuznyetsov** [comment]
@TylerJewell 
Now I'm doing final checks for this PR. Also, it turned out that purpose of aliases is not to support old configuration, but old code, that relies on old configuration.
So, here's how it is working:
- Only new property names will be allowed to be used in configuration (conf. files, system properties, env. variables).
- All our code will be also using new properties.
- Aliases are used in case, if someones old code is using old property names.

About those other PRs, I'd like to know if they're are final and ready to merge, and who will be merging them. I see that it should not conflict with this PR. (I don't see properties, that I've renamed).

**Comment by TylerJewell** [comment]
@mkuznyetsov - I see your questions on who will merge the other PRs.  Please connect with your TL and identify someone (maybe you?) who shoudl merge those PRs.  I wrote them both, and I have done personal testing, but now we need someone in the core team to review, comment, merge, test, and verify.  I supsect that we will break some of our CI systems when we do this, so we will need very good testing of che-launcher, che cli, and che-server.

We should do this at the beginning of the next sprint, as this PR will be merged at the beginning of M7.

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties
Comment (skabashnyuk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties]: new line

**Review by skabashnyuk — APPROVED** [review]

**Pull request merged**
